controllers/shipwrightbuild_controller.go

```diff
@@ -10,22 +10,28 @@ import (
 	"os"
 	"path/filepath"
 
-	mfc "github.com/manifestival/controller-runtime-client"
-	"github.com/manifestival/manifestival"
-
-	"github.com/go-logr/logr"
+	corev1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/errors"
 	"k8s.io/apimachinery/pkg/runtime"
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 
+	buildcorev1alpha1 "github.com/shipwright-io/build/pkg/apis/core/v1alpha1"
+	"github.com/shipwright-io/build/pkg/ctxlog"
 	"github.com/shipwright-io/operator/api/v1alpha1"
+
+	mfc "github.com/manifestival/controller-runtime-client"
+	"github.com/manifestival/manifestival"
+)
+
+const (
+	namespace string = "namespace"
+	name      string = "name"
```

Comment:
This seems unnecessary. Is there a particular reason that the actual strings can not be used instead of these extra variables?

Reply:
No reason, just to e conform to what it was done here https://github.com/shipwright-io/build/blob/main/pkg/reconciler/build/controller.go#L31